pkg/client/rules.go

```diff
@@ -32,18 +47,31 @@ func (r *CortexClient) CreateRuleGroup(ctx context.Context, namespace string, rg
 
 // DeleteRuleGroup creates a new rule group
 func (r *CortexClient) DeleteRuleGroup(ctx context.Context, namespace, groupName string) error {
+	path := rulerAPIPath
+	if r.legacy {
+		path = legacyAPIPath
+	}
 	escapedNamespace := url.PathEscape(namespace)
 	escapedGroupName := url.PathEscape(groupName)
+	path += "/" + escapedNamespace + "/" + escapedGroupName
```

Comment:
This can be simplified if the path prefix is stored as a string in the client struct.

Reply:
I agree. I'll simplify it.